src/pages/questions/QuestionDisplay.jsx

```diff
@@ -58,17 +57,49 @@ const getValueTagExamplesURL = (question) => {
   return "";
 };
 
-const QuestionDisplay = ({ question, answerQuestion }) => {
+const QuestionDisplay = ({ question, answerQuestion, resetFilters }) => {
   const { t } = useTranslation();
   const valueTagQuestionsURL = getValueTagQuestionsURL(question);
   const valueTagExamplesURL = getValueTagExamplesURL(question);
 
+  React.useEffect(() => {
+
+    function handleShortCut(event) {
+      if (question) {
```

Comment:
You should also verify if the event comes from an `inputè element or not

Here is an example in which I press K in an input, so it adds a "k" to the input value (the expected behavior) but it also skip the question (not expected behavior)



https://user-images.githubusercontent.com/45398769/180067577-f36ab90c-35a9-4a22-a29c-8cf80db05372.mp4

Here is a potential solution

https://stackoverflow.com/questions/7230976/javascript-event-handler-on-body-but-not-on-input